ci-operator/step-registry/addon/aws/osd/addon-aws-osd-workflow.yaml

```diff
@@ -0,0 +1,26 @@
+workflow:
+  as: addon-aws-osd
+  steps:
+    env:
+      API_HOST: stage
+      CHANNEL_GROUP: candidate
+      PARALLEL: "true"
+      MULTI_AZ: "true"
+      ENABLE_AUTOSCALING: "false"
+      MIN_REPLICAS: "3"
+      MAX_REPLICAS: "6"
```

Comment:
why do we need this and not just `REPLICAS`?

Reply:
removed; not using autoscaling so it's redundant